5.5.0.1 is broken due to multiple definition of 'connect_timeout'

The offending commit is https://github.com/linuxdeepin/go-lib/commit/0c8bc92cfb77ec9cc4e180f5a72a2f599f7d61c5

```
# pkg.deepin.io/lib/pulse
/usr/bin/ld: $WORK/b163/_x002.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x003.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x004.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x005.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x006.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x007.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x008.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x009.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x010.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x011.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x012.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x013.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x014.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x015.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
/usr/bin/ld: $WORK/b163/_x016.o:(.bss+0x0): multiple definition of `connect_timeout'; $WORK/b163/_x001.o:(.bss+0x0): first defined here
```

--- a/pulse/dde-pulse.h
+++ b/pulse/dde-pulse.h
@@ -40,7 +40,7 @@ DECLARE(card);
 DECLARE(module);
 DECLARE(sample);
 
-int connect_timeout;
+extern int connect_timeout;
 
 void _get_server_info(pa_threaded_mainloop*, pa_context *c, int64_t cookie);
 

